README.md

```diff
@@ -1,2 +1,4 @@
 # ssi-dim-wallet-stub
 Stub application for DIM Wallet
+
```

Comment:
even tough the readme is still WIP, I think it would be good to directly explain the abbreviation at least once, so `Stub application for Decentral Identity Management (DIM) Wallet`

Reply:
Addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/eca1968a1351659f0f66cb3d0c285cd1640724bd